Library/Homebrew/build.rb

```diff
@@ -105,7 +105,8 @@ def install
 
     formula.extend(Debrew::Formula) if ARGV.debug?
 
-    formula.brew do
+    formula.brew do |f, stg|
```

Comment:
Yeah, `stg` for "stage", as in "staging context"; the context that goes with `stage`-like methods. I didn't want to take up a whole argument slot with just the temporary directory; I think it should apply to staged temporary data besides just the temp directory. If `brew` or `stage` ended up creating other stuff, like database entries or receipt files or ephemeral logs in another directory, `preserve` could also apply to those. Don't think spelled-out `stage` can be used because a `stage` method might be called inside these.

`stage_context`, `stage_data`, `stage_files`? `tmp_data`, `tmp_context`, maybe?